.pylintrc

```diff
@@ -3,4 +3,4 @@ disable = C0330, C0326, locally-disabled, fixme
 
 
 [format]
-max-line-length = 88
```

Comment:
We had 88 before since we use black here (https://black.readthedocs.io/en/stable/compatible_configs.html)

Reply:
makes sense - I just changed it because I noticed that it failed on a line I didn't touch

I've fixed it now + added pre-commit to CI